src/app/tests/suites/TestBasicInformation.yaml

```diff
@@ -214,7 +214,7 @@ tests:
       attribute: "MaxPathsPerInvoke"
       response:
           # For now all-clusters-app only supports 1 max paths per invoke.
-          value: 1
+          value: 5
```

Comment:
Should just have a condition of >= 1, otherwise if you rebuild with different values, it may fail CI in ways that always require this to be changed.

Also, you need to regen YAML tests for this test to actually apply under Darwin.

Reply:
Regen is not needed. This yaml test is a CI only yaml test from my understanding and is run only against the linux all-clusters app